probe-rs/src/bin/probe-rs/cmd/info.rs

```diff
@@ -25,6 +25,19 @@ use crate::util::common_options::ProbeOptions;
 pub struct Cmd {
     #[clap(flatten)]
     common: ProbeOptions,
+    #[arg(long, value_parser = parse_hex)]
```

Comment:
It would be nice to explain what this is for the users and that this is SWD-only (if I understand it correctly).

Reply:
JTAG already supports multiple targets, so it's not necessary there. It is SWD only. I've added a comment.